packages/docs/zh/guide/advanced/router-view-slot.md

```diff
@@ -46,11 +44,11 @@ We can also use KeepAlive inside a Transition:
 </router-view>
 ```
 
-For more information about using RouterView with the Transition component, see the [Transitions](./transitions) guide.
+关于更多 RouterView 组件和 Transition 组件之间的互动，请参考 [Transitions](./transitions) 指南。
 
-## Passing props and slots
+## 传递 props 和插槽
 
-We can use the slot to pass props or slots to the route component:
+我们可以用插槽来传递 props 或 slots 给路由组件：
```

Comment:
After check , I find here yet another `slots`, should it translate? I have no idea.

Reply:
Yes, I think so. No worries. Let me do this.